packages/node_modules/@webex/plugin-meetings/test/unit/spec/locus-info/infoUtils.js

```diff
@@ -35,14 +35,14 @@ describe('plugin-meetings', () => {
           policy: {HINT_3: true},
           moderator: {HINT_1: true, HINT_2: true},
           coHost: {HINT_4: true},
-          userDisplayHints: ['HINT_3', 'HINT_1', 'HINT_2']
+          userDisplayHints: ['HINT_3', 'HINT_1', 'HINT_2', 'LOWER_SOMEONE_ELSES_HAND']
         });
 
         assert.deepEqual(InfoUtils.parse(info, ['COHOST']), {
```

Comment:
suggestion: I'd like to see a case where there are multiple roles (specifically both cohost and moderator) to ensure that the LOWER_SOMEONE_ELSES_HAND display hint only ends up in the userDisplayHints array once.